src/context/WalletProvider/NativeWallet/components/NativeImport.tsx

```diff
@@ -48,6 +48,7 @@ export const NativeImport = ({ history }: RouteComponentProps) => {
               size='lg'
               autoComplete='off'
               autoCorrect='off'
+              textTransform='lowercase'
```

Comment:
Please have a look at [text-transform docs](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform). This only transforms text to be *displayed* in a certain way and doesn't change its primitive string value.

As a result, this still produces an error with the presence of uppercase letters.